<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1030</width>
    <height>680</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>pathFinder</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>1030</width>
     <height>600</height>
    </size>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1021</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbig">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>562</y>
      <width>72</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string> 放大</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushlittle">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>562</y>
      <width>68</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>缩小</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTransfer">
    <property name="geometry">
     <rect>
      <x>834</x>
      <y>603</y>
      <width>94</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>普通查找</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserRoute">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>1021</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>603</y>
      <width>94</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>随机选择</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>603</y>
      <width>331</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout4" stretch="0,9,9">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="labelStart">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <property name="text">
        <string>救援点</string>
       </property>
       <property name="buddy">
        <cstring>StartRoad</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="StartRoad">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <item>
        <property name="text">
         <string>路网编号</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="StartPoint">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <item>
        <property name="text">
         <string>节点选择</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>600</y>
      <width>331</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout5" stretch="0,9,9">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="labelDestination">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <property name="text">
        <string>事故点</string>
       </property>
       <property name="buddy">
        <cstring>DstRoad</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="DstRoad">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <item>
        <property name="text">
         <string>路网编号</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="DstPoint">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <item>
        <property name="text">
         <string>选择节点</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushshowall">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>562</y>
      <width>68</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>全图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>562</y>
      <width>111</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>加载另一地图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>560</y>
      <width>111</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>还原默认地图</string>
    </property>
   </widget>
  </widget>
  <action name="actionQueryInfo">
   <property name="text">
    <string>(预留)</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
